eleven-group/pickup
# Feature/17

TO TEST :
every single behaviour that seems you weird on the form, let's try any error on any field (actually we have no verification on email in front, side, i'll do it asap)
you have to make down then make install your backend to make it works
HAVE FUN GUYS 


HOW TO TEST :
every required field should display an error if empty
if you enter a bad adress, a specific error should be raised for it
you can't have two times the same email address
should be redirected to confirmation page after submit
